client/packages/system/src/Patient/Components/PatientSearchModal/PatientSearchModal.tsx

```diff
@@ -0,0 +1,48 @@
+import React, { FC } from 'react';
+import {
+  createQueryParamsStore,
+  ListSearch,
+  QueryParamsProvider,
+  useTranslation,
+} from '@openmsupply-client/common';
+import { PatientRowFragment, usePatient } from '../../api';
+import { getPatientOptionRenderer } from '../PatientOptionRenderer';
+import { filterByNameAndCode, PatientSearchModalProps } from '../../utils';
+
+const PatientSearchComponent: FC<PatientSearchModalProps> = ({
+  open,
+  onClose,
+  onChange,
+}) => {
+  const { data, isLoading } = usePatient.document.list({
```

Comment:
Using usePatient hook instead of useNames hook because we have PatientQuery permission and there's a new PatientRepository in the BE...